Src/ILGPU/IR/Values/Terminators.cs

```diff
@@ -362,6 +387,21 @@ protected ConditionalBranch(in ValueInitializer initializer)
 
         #region Methods
 
+        /// <summary>
+        /// Returns true if this branch offers two branch targets and one of the
+        /// targets is the one provided.
+        /// </summary>
+        /// <param name="block">The branch target to exclude.</param>
+        /// <param name="otherBlock">The other branch target (if any).</param>
+        public bool TryGetOtherBranchTarget(BasicBlock block, out BasicBlock otherBlock)
+        {
+            otherBlock = null;
+            if (NumTargets > 2)
+                return false;
+            otherBlock = Targets[0] == block ? Targets[1] : Targets[0];
+            return true;
```

Comment:
QUESTION: Is this correct? The description says get the other branch if `block` is one of the targets.
```suggestion
            if (Targets[0] == block)
            {
                otherBlock = Targets[1];
                return true;
            }
            else if (Targets[1] == block)
            {
                otherBlock = Targets[0];
                return true;
            }
            return false;
```